bayesian_model/processor.py

```diff
@@ -0,0 +1,145 @@
+import os
+from lxml import etree
+import pickle as pkl
+
+from taggers import get_tagging_from_krnnt, get_tagging_from_morphoDita, get_tagging_from_wcrft2
+from utils.converter import convert_krnnt_output_file_to_xml
+
+PATH_TO_RAW_FILES = "../data/wikipedia_data"
+MORPHODITA_FOLDER = "morphoDita"
+KRNNT_FOLDER = "krnnt"
+WCRFT2_FOLDER = "wcrft2"
+
+
+def process_training_data_to_xmls(path_to_data: str):
```

Comment:
Why create separate methods for test and train?
```suggestion
def process_data_to_xmls(path_to_data: str, dataset_part: str):
```

Reply:
Commited in https://github.com/Ceglowa/ajp_1_dratwinski_gramacki/pull/4/commits/e4191758dd15ad239eb980cf79a4b1b567953baf